index.js

```diff
@@ -17,7 +19,7 @@ function isMobile (opts) {
   }
   if (typeof ua !== 'string') return false
 
-  var result = opts.tablet ? tabletRE.test(ua) : mobileRE.test(ua)
+  var result = mobileRE.test(ua) || (!!opts.tablet && tabletRE.test(ua))
```

Comment:
```suggestion
  var result = mobileRE.test(ua) || (opts.tablet && tabletRE.test(ua))
```

This boolean cast doesn't change the results

Reply:
Right!